### ISSUE TITLE
Key Press in iOS

### ISSUE THREAD

**Issue by ashishranjanJN** [appium/appium#12303]
Hi, I am working on Appium where I am stuck in one scenario where I want to enter some text say "ABC" using key press in iOS.
I know there is a way to do in Android i.e.
driver.pressKeyCode(AndroidKeyCode.A);
driver.pressKeyCode(AndroidKeyCode.B);
driver.pressKeyCode(AndroidKeyCode.C);

I wanted to know if there is any way in Appium wherein I can for iOS.

## Environment

* Appium Version 1.11.0 (1.11.0)
* iOS Real Device version 12.1

**Comment by dpgraham** [comment]
You'll need to use [Send Keys](http://appium.io/docs/en/commands/element/actions/send-keys/) to send keys to an element

**Comment by ashishranjanJN** [comment]
@imurchie Is is possible to pass Ket Event in iOS

**Comment by mykola-mokhnach** [comment]
XCTest does not provide any primitives to emulate sending keys like it is done in Android. Although you could access the on-screen keyboard keys there and perform clicks on them.

**Comment by ashishranjanJN** [comment]
@mykola-mokhnach But for that I need to highlight the textbox first.

I have this textbox in a WebKit which is loaded in an iOS Activity

**Comment by mykola-mokhnach** [comment]
Most likely. I don't know other ways to achieve that

**Comment by ashishranjanJN** [comment]
@dpgraham I have a different kind of situation here to automate.
I have achieved this in Android via Key Press wherein I have to enter text in WebView which is loaded in Native Activity.

I am struggling to achieve it in iOS.

I have tried:

driver.getKeyboard().pressKey(Keys.A);

But it gave error as:
org.openqa.selenium.UnsupportedCommandException: sendKeysToActiveElement
.
.
at org.openqa.selenium.remote.RemoteKeyboard.pressKey(RemoteKeyboard.java:46)
